src/button/notification.scss

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-button-notification($status: primary) {
+  position: relative;
+
+  &::before {
+    content: ' ';
+    position: absolute;
+    width: $iui-s;
+    height: $iui-s;
+    border-radius: 100%;
+    @include themed {
+      background-color: t(iui-color-foreground-#{$status});
+      border: 2px solid t(iui-color-background-1);
+    }
+    @media (prefers-reduced-motion: no-preference) {
+      transition: border-color $iui-speed-fast ease-out;
```

Comment:
Isn't transition also needed for background color?

Reply:
I guess if we had the dot change from green to red then yes, we would want the background-color to transition as well.  I can add that.